src/sortStudents.ts

```diff
@@ -1,16 +1,67 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' |'desc';
 
+function countSum(numbers: number[]): number {
+  return numbers.reduce((acc: number, num: number) => {
+    return acc + num;
+  }, 0);
+}
+
+export function sortStudents(
+  students: Student[], sortBy: SortType, order: SortOrder,
+): Student[] {
+  const copyArray = [...students];
+  const copyObject = copyArray.map((person) => ({ ...person }));
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      copyObject.sort((prev, current) => {
```

Comment:
```suggestion
      return copyObject.sort((prev, current) => {
```
You can return a result immediately.